src/main/resources/mustache-templates/search.html

```diff
@@ -36,49 +32,21 @@ <h4>Patient Search</h4>
 </div>
 
 <div class="container">
-	<ul id="patient_list">
-	</ul>
-	<p id="no-results" hidden="true">No Results Found</p>
+	{{#results}}
+		<h3>Search Results</h3>
+		<small class="text-muted">
+		Click on the patient to see their labs and HPO terms.</small>
+		<div class="vertical-space">
+			{{^patients}}
+				<p>No Results Found</p>
+			{{/patients}}
+			{{#patients}}
+				<div class="row" style="margin-left:inherit;">
```

Comment:
I'm overriding margin-left here because the "row" class pushed the links 15px left of the rest of the content. Better way to do this?
<img width="1320" alt="screenshot" src="https://user-images.githubusercontent.com/2576491/43662418-be8577d0-971a-11e8-825d-e0d2904cbb11.png">

Reply:
Does this need to be a 'row'? It might make more sense semantically to make this an unordered list and use a class to set the list-style-type to none.

Reply:
That is, something like this:

```
<div class="vertical-space">
  <ul class="list-unstyled">
  {{^patients}}
    <li>No Results Found</li>
  {{/patients}}
  {{#patients}}
    <li><a href="...">...</a></li>
  {{/patients}}
  </ul>
 </div>
```

Reply:
Yes - list makes more sense. It actually looks good without applying any class to it at all.